doc/sources/conf.py

```diff
@@ -128,7 +128,7 @@
 
 html_theme_options = {
     "logo_only": False,
-    "display_version": True,
+    "version_selector": True,
```

Comment:
Maybe it would be better to remove it at all? Because:

1.  Looks like `version_selector` won't work correctly anyway as our docs are not hosted at Read the Docs,
2. Our docs are not organized in the way to provide information for different versions. The information is only provided for the latest version anyway.

Reply:
I think it refers to this feature, which is working correctly in _some_ versions of the docs:
![2024-10-28 13 03 16 intel github io b05110fb1274](https://github.com/user-attachments/assets/59e26ef3-a422-475c-ba2a-e31beb9ee8cf)

Reply:
It actually shows more entries if you click it:
![image](https://github.com/user-attachments/assets/fa49c328-e104-4587-bddc-b0181e2392d2)